front/components/assistant/AssistantDetails.tsx

```diff
@@ -35,7 +35,11 @@ import { timeAgoFrom } from "@app/lib/utils";
 type AssistantDetailsProps = {
   owner: WorkspaceType;
   show: boolean;
-  onClose: () => void;
+  onClose: ({
+    shouldMutateAgentConfigurations,
+  }: {
+    shouldMutateAgentConfigurations: boolean;
+  }) => void | (() => void);
```

Comment:
```suggestion
  onClose: ({
    shouldMutateAgentConfigurations,
  }: {
    shouldMutateAgentConfigurations? : boolean;
  } = {}) => void);
```

Reply:
apparently can't be done on the type declaration